.gitignore

```diff
@@ -4,6 +4,9 @@
 ### Linux ###
 *~
 
+### VSCode ###
+.vscode
```

Comment:
We should not ignore the VSCode, it helps us that have project settings and the need to setup the VSCode will be no longer needed.